pkg/api/router.go

```diff
@@ -258,12 +255,10 @@ func (s *Service) mountAPI() {
 	)
 
 	handle("/pss/subscribe/{topic}", web.ChainHandlers(
-		s.gatewayModeForbidEndpointHandler,
```

Comment:
Here we remove the gateway forbidden middleware which blocks users from requesting the tags of all uploads, including others'. Has this been thoroughly discussed ?